pkg/reconciler/nscert/nscert_test.go

```diff
@@ -60,14 +61,13 @@ var (
 func newTestSetup(t *testing.T, configs ...*corev1.ConfigMap) (
 	ctx context.Context,
 	cancel context.CancelFunc,
-	rclr *reconciler,
+	controller *controller.Impl,
```

Comment:
Rather than changing the signature, can we change the `cancel` that we return to do what you are inlining elsewhere?